src/bindings/cpp/include/keyset.hpp

```diff
@@ -576,6 +591,7 @@ inline KeySet::~KeySet ()
  * If you don't want destruction of keyset at
  * the end you can release the pointer.
  * */
+// TODO (kodebach): remove?
```

Comment:
I think the `release()` function should be removed. Its original purpose is better solved by using the new `ksBorrow()` and its implementation is pretty hacky and makes a mostly useless allocation of a new KeySet.